src/linker.cpp

```diff
@@ -376,16 +376,24 @@ gb_internal i32 linker_stage(LinkerData *gen) {
 								LIT(obj_file),
 								LIT(build_context.extra_assembler_flags)
 							);						
+							if (!result) {
+								gb_printf_err("executing `nasm` to assemble foreing import of %.*s failed.\n\tSuggestion: `nasm` does not ship with the compiler and should be installed with your system's package manager.\n", LIT(asm_file));
```

Comment:
We probably should ship with it because its licence is compatible with the Odin compiler. But this is a good message for the time being.